ChangeLog.d/8824.txt

```diff
@@ -0,0 +1,6 @@
+Bugfix
+   * Fix mbedtls_pk_sign(), mbedtls_pk_verify(), mbedtls_pk_decrypt() or
+     mbedtls_pk_encrypt() on non-opaque RSA keys to honor the padding mode in
+     the RSA context. Before they always used PKCS#1 v1.5 even when the RSA
+     context was configured for PKCS#1 v2.1 (PSS/OAEP). Fixes #8824.
```

Comment:
Thanks for improving the format, but I think the description of the old behaviour is incomplete: the issue was only when MBEDTLS_USE_PSA_CRYPTO was enabled. Suggestion: "Before, if MBEDTLS_USE_PSA_CRYPTO was enabled, they always used..."

Reply:
Oops...